flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/NtTable.java

```diff
@@ -0,0 +1,91 @@
+package com.flash3388.frc.nt;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+public class NtTable {
+
+    private final NetworkTable mTable;
+
+    public NtTable(NetworkTable table) {
+        mTable = table;
+    }
+
+    public NtTable(String name) {
+        this(NetworkTableInstance.getDefault(), name);
+    }
+
+    public NtTable(NetworkTableInstance tableInstance, String name) {
+         this(tableInstance.getTable(name));
+    }
+
+    public boolean getBooleanEntryValue(String name) {
+        return getEntryValue(name).getBoolean();
+    }
+
+    public int getIntEntryValue(String name) {
+        return (int) getEntryValue(name).getDouble();
+    }
+
+    public double getDoubleEntryValue(String name) {
+        return getEntryValue(name).getDouble();
+    }
+
+    public String getStringEntryValue(String name) {
+        return getEntryValue(name).getString();
+    }
+
+    public NetworkTableEntry getEntry(String name) {
+        return mTable.getEntry(name);
+    }
+
+    private NetworkTableValue getEntryValue(String name) {
+        return getEntry(name).getValue();
+    }
+
+    public class Builder {
+
+        private final NetworkTable mTable;
+
+        public Builder(NetworkTable table) {
+            mTable = table;
+        }
+
+        public Builder(String name) {
+            this(NetworkTableInstance.getDefault(), name);
+        }
+
+        public Builder(NetworkTableInstance tableInstance, String name) {
```

Comment:
This should be second

Reply:
What's the convention exactly?

Reply:
Class cascade upwards

Reply:
Is it like "used method below the user method" but wrong and reverse?

Reply:
lol wrote without refreshing, but still got it

Reply:
How can something cascade upwards?

Reply:
Gravity